src/product-variant-stub/entities/product-variant-stub.entity.ts

```diff
@@ -0,0 +1,13 @@
+import { ObjectType, Field, Directive } from '@nestjs/graphql';
+import { UUID } from 'src/shared/scalars/CustomUuidScalar';
+import { v4 as uuidv4 } from 'uuid';
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+
+@ObjectType({ description: 'A product Item of a product variant' })
+@Schema()
+export class ProductVariantStub {
```

Comment:
And I am against this type of review.
Please provide more reasoning for change requests in the future.
It is an incomplete version of the product variant Entity with the field I need.
If you prefer "partial" I can change to that

Reply:
what do you exactly mean with incomplete?
that it has only the id field?
imho, from the point of view of your service, it is complete, as it contains all fields you require in this service
or do you have another ProductVariant (database type) which contains more information?

Reply:
Wait, are you telling me this is a GraphQL type?

Reply:
Ah, okay, I think I might understand what you mean.
However, if what I expect is true, I see even less reasons as to why the `incomplete` should be a part of the name:
The only information that matters for `inventory` is what it needs, so as far as `inventory` is concerned, this product variant is the full product variant no matter if it might have additional fields or not.

As stated above, I can accept it if there is a technical reason for the name such as the GraphQL schema getting a type conflict.
However, if there is no such reason, I'd like to avoid unreadable names (-> everything that has more parts than necessary).

Reply:
Yes indeed I have a  GraphQL type called ProductVariant which would lead to naming conflicts with the seperate collection. Therefore I need some kind of appendix. My proposals are:

- Stub
- Partial
- id